bin/apisix

```diff
@@ -339,6 +355,12 @@ function _M.start(...)
     init(...)
     init_etcd(...)
 
+	local need_version="1.15.8";
+	if not checkOpenrestyVersion(need_version) then
+		print("the openresty version must >=", need_version, "\n")
```

Comment:
should use `io.stderr:write`, please take a look at here:
https://github.com/iresty/apisix/blob/master/bin/apisix#L293